src/components/tool/guide/Instructions/AccessInstructions.js

```diff
@@ -17,7 +17,7 @@ export function AccessInstructions({ service, port, node, username }) {
           {username === "" && (
             <InstructionText>
               <i>
-                Replace <Code>qutusername</Code> with your QUT username.
+                Enter <Code>username</Code> with your QUT username.
```

Comment:
"Enter" here doesn't work grammatically. I think "Replace" is fine.

Reply:
It's empty with just a placeholder and you can't even highlight and replace the placeholder value though?

Reply:
![image](https://github.com/user-attachments/assets/049e4759-0d1d-401b-8b8c-85d37f6dbc73)

![image](https://github.com/user-attachments/assets/04cd56cb-b421-4377-bcc5-6c9eaba56507)

In the context it appears, it's referring to the command on the subsequent line. "Replace" makes sense because the user has not provided a username, so they have to replace "username"/"qutusername" with their user when they run the command.

Reply:
Sold!